CHANGELOG.md

```diff
@@ -13,6 +14,7 @@ Changes
 * [UI]: Fixed bug when creating/modifying a metadata template the user could not search for a field. (20.05.1)
 * [UI]: Fixed bug where managers could not load Users listing page. (20.05.1)
 * [UI]: Added functionality for users to synchronize assembly data from remote APIs.
+* [UI]: Removed gravatar and replaced with user icon.
```

Comment:
Wasn't the plan to use this library as a replacement? https://ant.design/components/avatar/

Reply:
It will be eventually, I just wanted to get gravatar out in a simple merge request.  I have another branch I am working on that is a full menu bar re-write, but it won't be ready for a little bit.  This is a baby step.  The full replacement bar should be done by the next release, but that will be big enough.